js/attmpt/signupAttmpt.js

```diff
@@ -0,0 +1,51 @@
+import id from "../email/idEvent.js"
+import pw from "../password/pwEvent.js";
+
+
+
+
+async function emailCheck(email) {
+  try{// POST 요청 보내기
+  const response = await fetch('https://bootcamp-api.codeit.kr/api/check-email', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({email : email})
+  })
+const data = await response.json();
+return Set.data
+}catch(error){
+  console.log(error);
+}
```

Comment:
r: indent가 다른 부분이 있네요
```suggestion
  const data = await response.json();
    return Set.data
  } catch(error) {
    console.log(error);
  }
```